binancial/data/get_klines_realtime.py

```
@@ -48,30 +71,57 @@ def handle_socket_message(msg):
+    
+    try:
+        # Open file with context manager
+        with open(file_path, 'a') as file:
+            logger.info(f"Starting kline stream for {symbol} with interval {interval}")
+            stream.start()
+            
+            # Store socket id for proper cleanup
+            socket_id = stream.start_kline_socket(
+                callback=handle_socket_message,
+                symbol=symbol,
+                interval=interval
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)